rules/windows/credential_access_mimikatz_powershell_module.toml

```diff
@@ -71,19 +77,43 @@ mean time to respond (MTTR).
 
 ## Config
 
-If enabling an EQL rule on a non-elastic-agent index (such as beats) for versions <8.2, events will not define `event.ingested` and default fallback for EQL rules was not added until 8.2, so you will need to add a custom pipeline to populate `event.ingested` to @timestamp for this rule to work.
+The 'PowerShell Script Block Logging' logging policy must be configured (Enable).
+
+Steps to implement the logging policy with with Advanced Audit Configuration:
+
+```
+Computer Configuration > 
+Administrative Templates > 
+Windows PowerShell > 
+Turn on PowerShell Script Block Logging (Enable)
+```
+
+Steps to implement the logging policy via registry:
+
+```
+reg add "hklm\\SOFTWARE\\Policies\\Microsoft\\Windows\\PowerShell\\ScriptBlockLogging" /v EnableScriptBlockLogging /t REG_DWORD /d 1
+```
 """
-references = ["https://attack.mitre.org/software/S0002/"]
-risk_score = 99
+references = [
+  "https://attack.mitre.org/software/S0002/",
+  "https://raw.githubusercontent.com/EmpireProject/Empire/master/data/module_source/credentials/Invoke-Mimikatz.ps1",
+]
+risk_score = 73
 rule_id = "ac96ceb8-4399-4191-af1d-4feeac1f1f46"
-severity = "critical"
+severity = "high"
 tags = ["Elastic", "Host", "Windows", "Threat Detection", "Credential Access"]
 timestamp_override = "event.ingested"
-type = "eql"
+type = "query"
 
 query = '''
-process where event.type in ("start", "process_started") and process.name : ("cmd.exe", "powershell.exe", "pwsh.exe")
-and process.args : ("*DumpCreds", "*Mimikatz*")
+event.category:process and
+powershell.file.script_block_text:(
+  (DumpCreds and
+  DumpCerts) or
+  "sekurlsa::logonpasswords" or
+  ("crypto::certificates" and
+  "CERT_SYSTEM_STORE_LOCAL_MACHINE")
```

Comment:
Thoughts on adding more invokes for mimikatz?

```
CRYPTO::Certificates
KERBEROS::Golden
KERBEROS::List 
KERBEROS::PTT
LSADUMP::DCSync
LSADUMP::LSA
LSADUMP::SAM
LSADUMP::Trust
MISC::AddSid
MISC::MemSSP 
MISC::Skeleton
PRIVILEGE::Debug
SEKURLSA::Ekeys
SEKURLSA::Kerberos
SEKURLSA::Krbtgt
SEKURLSA::LogonPasswords
SEKURLSA::Pth
SEKURLSA::Tickets
TOKEN::List 
TOKEN::Elevate
TOKEN::Elevate /domainadmin 
```

Reply:
On Invoke-Mimikatz (https://raw.githubusercontent.com/EmpireProject/Empire/master/data/module_source/credentials/Invoke-Mimikatz.ps1), these are the only occurrences of hardcoded mimikatz commands (in the code, the privilege::debug is mentioned in the description), other commands can be passed as arguments, but they will not be captured in Powershell logs, only on command line ones.

Reply:
Gotcha, thanks for explaining!